ci/tasks/light-aws/test-drivers.sh

```
@@ -0,0 +1,63 @@
+# Fixtures
+export S3_MACHINE_IMAGE_URL=${uploaded_machine_image_url}
+export S3_MACHINE_IMAGE_FORMAT=${uploaded_machine_image_format}
+export EBS_VOLUME_ID=${existing_volume_id}
+export EBS_SNAPSHOT_ID=${existing_snapshot_id}
+export AMI_FIXTURE_ID=${ami_fixture_id}
+export PRIVATE_AMI_FIXTURE_ID=${private_ami_fixture_id}
+
+echo "Downloading machine image"
+export MACHINE_IMAGE_PATH=${tmp_dir}/image.iso
+export MACHINE_IMAGE_FORMAT="RAW"
+wget -O ${MACHINE_IMAGE_PATH} http://tinycorelinux.net/7.x/x86_64/archive/7.1/TinyCorePure64-7.1.iso
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)